tests/app/main/views/test_users.py

```diff
@@ -283,3 +284,155 @@ def test_download_csv(self, data_api_client):
         response = self._return_user_export_response(data_api_client, framework, users)
         assert response.status_code == 200
         self._assert_things_about_user_export(response, users)
+
+
+@mock.patch('app.main.views.users.data_api_client')
+class TestBuyersExport(LoggedInApplicationTest):
+
+    def test_response_data_has_buyer_info(self, data_api_client):
+        data_api_client.find_users_iter.return_value = [
+            {
+                'id': 1,
+                "name": "Chris",
+                "emailAddress": "chris@gov.uk",
+                "phoneNumber": "01234567891",
+                "createdAt": "Thu, 04 Aug 2016 12:00:00 GMT"
+            }
+        ]
+
+        data_api_client.find_briefs_iter.return_value = [
+            {
+                'title': 'This is a brief',
+                'status': 'draft',
+                'users': [{
+                    'id': 1
+                }]
+            }
+        ]
+
+        response = self.client.get('/admin/users/download/buyers')
+        rows = [line.split(",") for line in response.get_data(as_text=True).splitlines()]
+        buyer = rows[1]
+
+        assert response.status_code == 200
+        assert buyer == [u'Chris', u'chris@gov.uk', u'01234567891',
```

Comment:
Could be good to assert the values of the header row too (probably just in one of your tests, no need for overkill).